tests/physics/controllers/coupled_motor/coupled_motor.c

```diff
@@ -170,7 +170,7 @@ int main(int argc, char **argv) {
       motors[i] = wb_robot_get_device_by_index(2 * i + 1);
 
     double positions[NB_SENSORS];
-    double tolerance = 1e-10;
+    double tolerance = 1e-4;
```

Comment:
I'm not sure to understand why this specific test required changing the tolerances and not others. As far as I recall (and it has been a while, so I might recall incorrectly), mostly we test that instructions provided to one motor (and relayed to the others in the coupling) result in both joints reaching the same position (at a multiplier coefficient difference, if a multiplier is present). So intuitively we should expect them to be very strictly close to each other. Was this change necessary?

Reply:
Yes, these tolerance changes are necessary. They reflect the amounts that the *real* joint positions have always varied when not under position control. We just didn't notice before because the position values reported were integrated estimates based on the joint velocities, not the real joint positions. Note that the tolerance for the position control portion of the test did not change. Make sense?